tests/pot/config.yml

```diff
@@ -0,0 +1,14 @@
+model:
+  model_ir_path : 'model_ir'

```

Comment:
I'm wondering if we can use cache directory to store IR files. Let me check and add update here.